test/test_model.py

```diff
@@ -1,49 +1,85 @@
+from typing import Tuple
+
 import pytest
 from pytest import fixture
 import torch
 import numpy as np
+
+from config import Config
 from model import Model
+from src.utils import load_data
+
 
 @fixture
 def input():
     np.random.seed(42)
     data = np.random.uniform(0, 10, 24).reshape(24, 1)
     return torch.Tensor(data)
 
+@fixture 
+def config():
+    return Config(
+        input_size=1,
+        output_size=1,
+        hidden_size=[],
+        learning_rate=0.005,
+        epochs=800,
+    )
+    
+@fixture
+def data_dims() -> Tuple[int, int]:
+    return (671, 1)
+
 @fixture
 def compare():
     np.random.seed(4242)
     data = np.random.uniform(0, 10, 24).reshape(24, 1)
     return torch.Tensor(data)
 
-@pytest.mark.usefixtures("input")
-def test_single_layer(input):
+@pytest.mark.usefixtures("input", "config")
+def test_single_layer(config: Config, input: torch.Tensor):
     """_summary_
 
     Args:
         input (Tensor): an Nx1 set of inputs
     """
-    #Construct the model, 1 input, 1 output
-    model = Model(1,1)
+    #Construct the model, 1 input, 1 output\
+    model = Model(config)
     #Pass input through the model
     tmp = model(input)
 
-@pytest.mark.usefixtures("input")
-def test_two_layer(input, compare):
+@pytest.mark.usefixtures("input", "config")
+def test_two_layer(config: Config, input, compare):
     """_summary_
 
     Args:
         input (_type_): _description_
         compare (_type_): _description_
     """
     #Construct the model
-    layers = [10]
-    model = Model(1,1, layers)
+    config.hidden_size = [10]
+    model = Model(config)
     #Pass input through the model
     tmp = model(input)
 
-@pytest.mark.usefixtures("input")
-def test_three_layer(input, compare):
+@pytest.mark.usefixtures("input", "config")
+def test_three_layer(config: Config, input, compare):
```

Comment:
Not sure why this is duplicated. Based on the print, I can guess maybe it was a dev/debug function?